ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParsePayload.java

```diff
@@ -31,8 +31,9 @@
  */
 public class ParsePayload extends MapElementsWithErrors.ToPubsubMessageFrom<PubsubMessage> {
 
-  public static ParsePayload of(ValueProvider<String> schemasLocation) {
-    return new ParsePayload(schemasLocation);
+  public static ParsePayload of(ValueProvider<String> schemasLocation,
+      ValueProvider<String> aliasingConfigurationLocation) {
+    return new ParsePayload(schemasLocation, aliasingConfigurationLocation);
```

Comment:
I'd prefer the variable names follow the option names all the way through, so I'd like to see `aliasingConfigurationLocation` change to `schemaAliasesLocation` throughout this PR.